Description: <Added DeskStretches image>
 Added exercise tab image file
 .
 breakmywork (1.0) stable; urgency=low
 .
   * Initial release (Closes: #nnnn)  <nnnn is the bug number of your ITP>
Author: Ravikiran Janardhana <ravikiran.j.127@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- breakmywork-1.0.orig/Makefile
+++ breakmywork-1.0/Makefile
@@ -10,6 +10,7 @@ install:
 		cp breakmywork.py $(BASEDIR)
 		chmod 755 $(BASEDIR)/breakmywork.py
 		cp images/breakicon.png $(IMGDIR)
+		cp images/deskStretches.jpg $(IMGDIR)
 		ln -s $(BASEDIR)/breakmywork.py $(BINDIR)/breakmywork
 		chmod 755 $(BINDIR)/breakmywork
 uninstall:
